pkg/analyzer/analyzers/shopify/expected_output.json

```diff
@@ -0,0 +1,219 @@
+{
+    "AnalyzerType":15,
+    "Bindings":[
+       {
+          "Resource":{
+             "Name":"My Store",
+             "FullyQualifiedName":"detectors@trufflesec.com",
+             "Type":"shop",
+             "Metadata":{
+                "created_at":"2024-08-16T17:16:17+05:00"
+             },
+             "Parent":null
+          },
+          "Permission":{
+             "Value":"Read",
+             "Parent":null
+          }
+       },
+       {
+          "Resource":{
+             "Name":"My Store",
+             "FullyQualifiedName":"detectors@trufflesec.com",
+             "Type":"shop",
+             "Metadata":{
+                "created_at":"2024-08-16T17:16:17+05:00"
+             },
+             "Parent":null
+          },
+          "Permission":{
+             "Value":"Read",
+             "Parent":null
+          }
+       },
```

Comment:
These bindings look exactly the same. Are we setting the permission correctly?

Reply:
It's also just "Read" which makes me think we're not using the generated permission values.

Reply:
thank you for pointing it out. I noted that permissions where being assigned to shop resource instead of categories. Fixed this issue.